drivers/backup/auth.go

```diff
@@ -213,12 +229,18 @@ func getKeycloakEndPoint(admin bool) (string, error) {
 }
 
 // GetPxBackupNamespace returns namespace of px-backup deployment.
-func GetPxBackupNamespace() string {
-	ns, present := os.LookupEnv(pxBackupNamespace)
-	if !(present) {
-		return AdminTokenSecretNamespace
+func GetPxBackupNamespace() (string, error) {
+	var allServices *corev1.ServiceList
+	var err error
+	if allServices, err = k8sCore.ListServices("", metav1.ListOptions{}); err != nil {
```

Comment:
nit: just remove defining vars for `err` and `allServices` and do
```
allServices, err := k8sCore.ListServices("", metav1.ListOptions{})
if err != nil {
    return "", fmt.Errorf("failed to get list of services. Err: %v", err)
}
```

Reply:
Done.